cli_helpers/tabular_output/terminaltables_adapter.py

```diff
@@ -25,4 +24,10 @@ def adapter(data, headers, table_format=None, **kwargs):
     table = table_format_handler[table_format]
 
     t = table([headers] + list(data), **filter_dict_by_key(kwargs, keys))
-    return t.table
+
+    dimensions = terminaltables.width_and_alignment.max_dimensions(
+        t.table_data,
+        t.padding_left,
+        t.padding_right)[:3]
+    for row in t.gen_table(*dimensions):
+        yield ''.join(''.join(row))
```

Comment:
Yes, "row" is a bit confusing because it's not a row; I've copied this from:
https://github.com/Robpol86/terminaltables/blob/master/terminaltables/build.py#L151

It is called by https://github.com/Robpol86/terminaltables/blob/master/terminaltables/base_table.py#L217

I'll have a look if I can replace this by `itertools.chain.from_iterable`